lib/components/album_image.dart

```diff
@@ -219,3 +219,75 @@ class _AlbumImageErrorPlaceholder extends StatelessWidget {
     );
   }
 }
+
+class TapToZoomImage extends StatefulWidget {
+  final AlbumImage albumImg;
+  final bool isDisabled;
+
+  const TapToZoomImage({
+    Key? key,
+    required this.albumImg,
+    this.isDisabled = false,
+  }) : super(key: key);
+
+  @override
+  _TapToZoomImageState createState() => _TapToZoomImageState();
+}
+
+class _TapToZoomImageState extends State<TapToZoomImage> {
+  @override
+  Widget build(BuildContext context) {
+    if (widget.isDisabled) {
+      return widget.albumImg;
+    }
+
+    return GestureDetector(
+      onTap: () => Navigator.of(context).push(
+        PageRouteBuilder(
+          opaque: false,
+          barrierDismissible: true,
+          transitionDuration: const Duration(milliseconds: 300),
+          pageBuilder: (
+            BuildContext context,
+            Animation<double> animation1,
+            Animation<double> animation2
+          ) {
```

Comment:
Can be blur the background here, or put a semi-transparent overlay behind the zoomed image?  

And is it possible to keep the original album image in place when opening the zoomed version, or does that break the hero transition? The empty space looks a bit weird.